lib/tcti/tpm2_tools_tcti_socket.c

```diff
@@ -0,0 +1,50 @@
+#include <inttypes.h>
+#include <stdlib.h>
+
+#include <tcti/tcti_socket.h>
+#include <sapi/tpm20.h>
+
+#include "log.h"
+#include "tpm2_util.h"
+
+#define TCTI_SOCKET_DEFAULT_ADDRESS "127.0.0.1"
+#define TCTI_SOCKET_DEFAULT_PORT     2321
+
+#define TPM2TOOLS_ENV_SOCKET_ADDRESS "TPM2TOOLS_SOCKET_ADDRESS"
+#define TPM2TOOLS_ENV_SOCKET_PORT    "TPM2TOOLS_SOCKET_PORT"
+
+TSS2_TCTI_CONTEXT*
+tpm2_tools_tcti_socket_init (char *opts)
+{
+    UNUSED(opts);
```

Comment:
Same comment here, this really shouldn't be unused but instead the hostname and port should be parsed since can be provided via --tcti-args